index.js

```diff
@@ -3,7 +3,7 @@
 const mimes = require('./mimes.json');
 
 function getMime(path) {
-  const extension = path.split('.').pop();
+  const extension = path.toLowerCase().split('.').pop();

```

Comment:
Would this make more sense?
`const extension = path.split('.').pop().toLowerCase()`
Iterating over the smaller string